### ISSUE TITLE
Tables: sorting by date does not work during DST

### ISSUE THREAD

**Issue by muling** [wet-boew/wet-boew#8386]
Table plugin seems to display the dates in the wrong order during the start of daylight savings time. The order is correct after 4pm.

![sortbug](https://user-images.githubusercontent.com/2365028/39822566-e53f63da-5378-11e8-8337-072bb2ecf55a.PNG)

No ordering has been applied to the dates using data-order.

**Comment by muling** [comment]
Can I email it to you as opposed to pasting it here?

**Comment by duboisp** [comment]
Yes, you can find my email on [GCCollab](https://gccollab.ca/profile/Pierre.Dubois) or on [GCConnex](https://gcconnex.gc.ca/profile/duboisp)

**Comment by muling** [comment]
Sent. Hope you got it.

**Comment by duboisp** [comment]
I got it, thanks. It's added to be work queue, I would try to take a quick look today or tomorrow.

**Comment by LiamLowndes** [comment]
I tried to reproduce the bug with the latest version of WET and the code sample you provided us but I was unsuccessful.

**Comment by muling** [comment]
so somehow got fixed in the later versions of WET?  We are using an older version of the template.

**Comment by duboisp** [comment]
@LiamLowndes can you test it in FF?

**Comment by duboisp** [comment]
@muling is there any error displayed in your FF and Chrome console?

**Comment by muling** [comment]
I didn't get any errors, just a couple of warnings.
The resource at “https://www.googletagmanager.com/gtm.js?id=GTM-TLGQ9K” was blocked because tracking protection is enabled.[Learn More]
real_time_e.html
Loading failed for the <script> with source “https://www.googletagmanager.com/gtm.js?id=GTM-TLGQ9K”.
real_time_e.html:1
